create_keepalived_sysext.sh

```diff
@@ -0,0 +1,121 @@
+  echo "The build process requires docker"
+  "${SCRIPTFOLDER}"/bake.sh --help
+  exit 1
+fi
+
+VERSION="$1"
+SYSEXTNAME="$2"
+
+if ! which docker &>/dev/null; then
+  echo Missing docker in path
+  exit 1
+fi
+
+SUFFIX=
+if  [ "${ARCH}" = "x86-64" ] || [ "${ARCH}" = "x86_64" ]; then
+  ARCH=amd64
+elif [ "${ARCH}" = "arm64" ]; then
+  ARCH="aarch64"
+  SUFFIX="v8"
+fi
+IMG=docker.io/"${ARCH}${SUFFIX}"/alpine:3.19
+mkdir -p "${SYSEXTNAME}"
+cat >"${SYSEXTNAME}"/build.sh <<EOF
+apk --no-cache add \
+        binutils \
+        file \
+        file-dev \
+        gcc \
+        glib \
+        glib-dev \
+        ipset \
+        ipset-dev \
+        iptables \
+        iptables-dev \
+        libmagic-static \
+        libmnl-dev \
+        libnftnl-dev \
+        libnl3-static \
+        libnl3-dev \
+        make \
+        musl-dev \
+        net-snmp-dev \
+        openssl \
+        openssl-dev \
+        openssl-libs-static \
+        pcre2 \
+        pcre2-dev \
+        autoconf \
+        automake zlib-static  alpine-sdk linux-headers libmnl-static git
+cd /opt
+git clone https://github.com/acassen/keepalived.git
+set -ex && \
+    cd /opt/keepalived && git checkout $VERSION && \
+    ./autogen.sh && \
+    CFLAGS='-static -s' LDFLAGS=-static ./configure  --disable-dynamic-linking \
+    --prefix=/usr \
+    --exec-prefix=/usr \
+    --bindir=/usr/bin \
+    --sbindir=/usr/sbin \
+    --sysconfdir=/usr/etc \
+    --datadir=/usr/share \
+    --localstatedir=/var \
+    --mandir=/usr/share/man \
+    --enable-bfd \
+    --enable-nftables \
+    --enable-regex \
+    --enable-json  --with-init=systemd --enable-vrrp --enable-libnl-dynamic
+set -ex && \
+    cd /opt/keepalived && \
+    make && \
+    make DESTDIR=/install_root install && \
+    find /install_root && \
+    rm -rf /install_root/usr/share /install_root/usr/etc/keepalived/samples && chown $(id -u):$(id -g) /install_root -R
+EOF
+chmod +x "${SYSEXTNAME}"/build.sh
+docker run -v "${SCRIPTFOLDER}"/"${SYSEXTNAME}":/install_root/  --rm -it alpine:3.19 /bin/sh -c /install_root/build.sh
+mkdir -p  "${SYSEXTNAME}"/usr/lib/systemd/system/
+cat > "${SYSEXTNAME}"/usr/lib/systemd/system/keepalived.service <<-'EOF'
+[Unit]
+Description=LVS and VRRP High Availability Monitor
+After=network-online.target syslog.target
+Wants=network-online.target
+
+[Service]
+Type=forking
+PIDFile=/run/keepalived.pid
+KillMode=process
+EnvironmentFile=-/usr/etc/sysconfig/keepalived
+EnvironmentFile=-/etc/sysconfig/keepalived
+ExecStart=/usr/sbin/keepalived $KEEPALIVED_OPTIONS
+ExecReload=/bin/kill -HUP $MAINPID
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+mkdir -p "${SYSEXTNAME}"/usr/lib/systemd/system/keepalived.service.d
+cat > "${SYSEXTNAME}"/usr/lib/systemd/system/keepalived.service.d/10-keepalived.conf <<-'EOF'
+[Service]
+ExecStartPre=/usr/bin/mkdir -p /etc/keepalived/ 
+ExecStartPre=-/bin/bash -c '[ ! -f /etc/keepalived/keepalived.conf ] && touch /etc/keepalived/keepalived.conf' 
```

Comment:
```suggestion
ExecStartPre=/bin/bash -c 'set -e; mkdir -p /etc/keepalived/; if ! [ -e /etc/keepalived/keepalived.conf ]; then touch /etc/keepalived/keepalived.conf; fi' 
```
Using `if` won't propagate the condition as return code and we can drop the `-` in `=-`